Chart.yaml

```
@@ -2,10 +2,10 @@ apiVersion: v2
 name: go-ratelimit
 description: A Ratelimit Service Helm chart for Kubernetes
 type: application
-version: 1.0.2
+version: 1.0.3
 appVersion: 1.5.0
 dependencies:
 - name: "redis-sharded"
-  version: 0.1.0
+  version: 0.3.0
   alias: "redish"
-  repository: "@softonic-public"
+  repository: "https://charts.softonic.io"
```

What do you think of adding a condition here to be able to disable the installation of the redis chart?
```suggestion
  repository: "https://charts.softonic.io"
  condition: redish.enabled
```